deploy/mpi-operator.yaml

```diff
@@ -145,6 +145,14 @@ rules:
   - mpijobs/status
   verbs:
   - "*"
+- apiGroups:
+  - scheduling.incubator.k8s.io
```

Comment:
Thanks for the PR. Could you explain why this is needed?

Reply:
I think it is to give the controller access to volcano's CRDs such as Queue and PodGroup to support running mpi-operator with volcano

Reply:
Okay sounds good. Are there any code changes required in order to support that?

Reply:
none